src/components/LocationSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Location } from '../types/sparkeats';
+
+type LocationSelectProps = {
+  locations: Location[];
+};
+
+function LocationSelect({ locations }: LocationSelectProps) {
+  const allCities = locations.map((location) => location.city);
+  const uniqueCities = [...new Set(allCities)];
+
+  return (
+    <div className="location-select">
+      <h2 className="location-select__title">Pick a Location</h2>
+
+      <select className="location-select__menu" name="location" id="location">
```

Comment:
> Most often the result is not fully accessible; browser and assistive technology is inconsistent; and implementations vary across the web. This is why it’s always better to use native selects with HTML/web.

I this is the right idea, but I disagree with the conclusion that it is __always__ better to use native select elements. I think it depends on goals, time, capabilities etc. Although accessibility usually a higher priority than design, I think we can also meet both needs by using both a native select and keeping the custom one to display. I found [this article](https://css-tricks.com/striking-a-balance-between-native-and-custom-select-elements/), which I think is a good overview of the rationale and approach.

This is potentially another reason to rebuild this component in #432.

What do you think?